testApi/src/main/kotlin/testApi/testRunner/TestRunner.kt

```diff
@@ -245,6 +246,9 @@ abstract class AbstractCoreTest {
         )
     }
 
+    fun externalDocumentationLink(url: String, packageListUrl: String? = null) =
```

Comment:
Note: The gradle maintenance branch will also include a very similar factory function. We should keep in mind to, in the end, just keep one of both